vars/abapEnvironmentPipeline.groovy

```diff
@@ -0,0 +1,51 @@
+void call(parameters) {
+    pipeline {
+        agent none
+        options {
+            skipDefaultCheckout()
+        }
+        stages {
+
+            stage('Init') {
+                steps {
+                    library 'piper-lib-os-dev'
```

Comment:
I guess this line needs to be removed.
Pipeline library is already loaded through the `Jenkinsfile` and in addition `piper-lib-os-dev` is no name which we support.